Userland/Libraries/LibGUI/FontPicker.cpp

```diff
@@ -57,6 +58,9 @@ FontPicker::FontPicker(Window* parent_window, const Gfx::Font* current_font, boo
     m_weight_list_view->set_model(adopt(*new FontWeightListModel(m_weights)));
     m_weight_list_view->horizontal_scrollbar().set_visible(false);
 
+    m_size_spin_box = widget.find_descendant_of_type_named<SpinBox>("size_spin_box");
+    m_size_spin_box->set_range(1, 36);
```

Comment:
A limit of 36 indeed seems rather arbitrary - a limit itself is probably a good idea but how about a higher (still arbitrary) number somewhere > 100?

@danboid what would you suggest?

Reply:
LO Writer seems to max out at 999.9 so maybe we should go with that too?

Reply:
It looks like font sizes greater than 255 are not supported by `LibGFX::Font` since `LibGFX::Font::presentation_size()` is defined to only return a `u8`.
This seems to cause the maximum safe font size to be 255 which is what I propose we set the maximum to.
Thoughts?

Reply:
255 will be fine. I've rarely used fonts greater than ~150 when doing WP/DTP sorta stuff so 255 should be fine for most cases.